CharlyUwampojejeCV.css

```diff
@@ -25,15 +27,15 @@ body {
     float: left;
     text-align: center;
     font-size: 90%;
-    padding: 20px 0px 50px 0px;
+    padding: 20px 20px 50px 20px;
```

Comment:
Please, take a look on my details in the ticket. I added some more details. This modification is not necessary.